sdk/keyvault/azure-security-keyvault-secrets/src/samples/README.md

```diff
@@ -23,9 +23,11 @@ This document describes how to use samples and what is done in each sample.
 
 ## Getting started
 
-Typically, you will not need to install or specifically depend on Azure Core, instead it will be transitively downloaded by your build tool when you depend on of the client libraries using it. In case you want to depend on it explicitly (to implement your own client library, for example), include the following Maven dependency:
+### Adding the package to your project
 
-[//]: # ({x-version-update-start;com.azure:azure-core;current})
+Maven dependency for Azure Secret Client library. Add it to your project's pom file.
+
+[//]: # ({x-version-update-start;com.azure:azure-security-keyvault-secrets;current})
```

Comment:
The Key Concepts section above is talking about Key Client and CryptoClient in secrets samples README.

Reply:
Should replace it with the Key Concepts section from Secrets Readme